docs/spec/v2/metadata.json

```diff
@@ -469,6 +477,51 @@
             }
           }
         },
+        "memory": {
+          "description": "Memory holds the memory of the system on which the monitored service is running, in bytes.",
+          "type": [
+            "null",
+            "integer"
+          ],
+          "minimum": 1
+        },
+        "network": {
+          "description": "Network holds information about the network that the system is connected to. This is only relevant for browsers and mobile devices.",
+          "type": [
+            "null",
+            "object"
+          ],
+          "properties": {
+            "downlink": {
+              "description": "Downlink holds the effective bandwidth estimate in megabits per second.",
+              "type": [
+                "null",
+                "integer"
+              ]
+            },
+            "effectivetype": {
+              "description": "EffectiveType describes the effective network type as one of a discrete set of network types, e.g. 'slow-2g', '2g', '3g', or '4g'.  See https://developer.mozilla.org/en-US/docs/Web/API/NetworkInformation for more details.",
```

Comment:
should we move this URL under `network` so all fields can be referenced from there.

Reply:
Yes, done.